staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -451,14 +451,31 @@ func describeNamespace(namespace *corev1.Namespace, resourceQuotaList *corev1.Re
 		printLabelsMultiline(w, "Labels", namespace.Labels)
 		printAnnotationsMultiline(w, "Annotations", namespace.Annotations)
 		w.Write(LEVEL_0, "Status:\t%s\n", string(namespace.Status.Phase))
+
+		if len(namespace.Status.Conditions) > 0 {
+			w.Write(LEVEL_0, "Conditions:\n")
+			w.Write(LEVEL_1, "Type\tStatus\tLastTransitionTime\tReason\tMessage\n")
+			w.Write(LEVEL_1, "----\t------\t------------------\t------\t-------\n")
```

Comment:
What about a quite long message or reason?

Reply:
Please, when leaving a review comment, explain what your concern is, and, if possible, give an example.

There are many functions in this package that use this same formatting. The writer interprets the tabs as separators, not as literal tabs. Here's what the output looks like with a long reasons and message:

```
Name:         example
        Labels:       <none>
        Annotations:  <none>
        Status:       Terminating
        Conditions:
          Type                             Status  LastTransitionTime               Reason                                                                                                                                                                                                             Message
          ----                             ------  ------------------               ------                                                                                                                                                                                                             -------
          NamespaceDeletionContentFailure  True    Wed, 15 Jan 2014 00:00:00 +0000  eexample reason example reason example reason example reason example reason example reason example reason example reason example reason example reason example reason example reason xample reason example reason  example message example message example message example message example message example message example message example message example message example message example message example message example message

        No resource quota.

        No LimitRange resource.
```

Reply:
thanks for your advices @dlipovetsky , the code lgtm.